app/models/article.rb

```diff
@@ -416,6 +416,19 @@ def access_by?(user)
     user.admin? || user_id == user.id
   end
 
+  def merge_with(merge_id)
+    begin
+      merge_art = Article.find(merge_id)
+      self.body += merge_art.body
+      self.comments += merge_art.comments
+      self.save
+      merge_art.delete
+      return true
+    rescue
+      return false
```

Comment:
If this method ultimately returns a success indicator, it might be good to name the method accordingly. Right now `merge_with` doesn't give me a lot of information